src/tribol/mesh/CouplingScheme.cpp

```diff
@@ -956,6 +962,8 @@ int CouplingScheme::apply( integer cycle, real t, real &dt )
 //------------------------------------------------------------------------------
 bool CouplingScheme::init()
 {
+   this->setSlicLoggingLevel();
```

Comment:
If we're changing the logging level, we should restore it at the end of the function.

Unfortunately, axom's `ScopedLogLevelChanger` has not yet been added to slic, 
(see: https://github.com/LLNL/axom/issues/1078)

Reply:
I think we should add something similar as a utility class until it gets into axom

See: https://github.com/LLNL/axom/blob/ca7e58ff86f5f16c4c1f57f3042bd911905adb13/src/axom/quest/interface/internal/QuestHelpers.hpp#L34-L61

Note: That class uses RAII, so it will undo itself when it goes out of scope, which might not be the right call here.

Reply:
This has been completely refactored and is controlled at the tribol API level. You set a logging level per coupling scheme. This seems most natural, but also flexible in that one could in principle debug a particular coupling scheme? Alternatively, we can set one logging level for ALL coupling schemes. Thoughts?

Reply:
I have modified things per our design discussion. We don't modify the logging for all coupling schemes (and tribol calls) in tribol::initialize() and let the host code control this. We 'can' modify logging per coupling scheme, but this will be enhanced in a future PR. The assumption is that the host-code handles the restoration of the logging level after tribol::update() and they handle all initialization and finalization. Enhancing this is out of the scope of this PR.